modules/manage/pages/raft-group-reconfiguration.adoc

```diff
@@ -0,0 +1,66 @@
+= Raft Group Reconfiguration in Redpanda
+:description: Learn how the Redpanda Raft group protocol provides consistency and availability during reconfiguration.
+
+Redpanda and Kafka with Raft (KRaft) both use the https://raft.github.io/[Raft consensus algorithm] to manage cluster metadata.Redpanda stores this metadata as messages in a special topic in a Raft cluster instance, also known as a Raft group. In a Raft group, one member is the leader and the others are followers. A Raft group that stores cluster metadata is called a controller, and the leader of this group is the controller broker. The messages in the metadata topic contain the current set of brokers that are members of the group. 
```

Comment:
```suggestion
Redpanda uses the glossterm:Raft[] consensus algorithm to manage cluster metadata. Redpanda stores this metadata as messages in a special topic in a Raft cluster instance, also known as a Raft group. In a Raft group, one member is the leader and the others are followers. A Raft group that stores cluster metadata is called a controller, and the leader of this group is the controller broker. The messages in the metadata topic contain the current set of brokers that are members of the group. 
```

Should we update our glossary definition to `controller broker`? https://raw.githubusercontent.com/redpanda-data/docs/shared/modules/terms/partials/controller.adoc

Reply:
@JakeSCahill I think it is a bit confusing that `controller` in this context can mean a "special metadata Raft group" and then the broker that is the current leader of the group is `controller broker`, but in general, `controller` refers to a single broker, and we'd just say `partition leader` (glossary term [here](https://docs.redpanda.com/current/reference/glossary/#partition-leader)) to refer to the Raft group leader. 

@mmaslankaprv is it possible to resolve this? Should we change `controller` to `controller broker` in the glossary? Or would it be better to remove `A Raft group that stores cluster metadata is called a controller, and the leader of this group is the controller broker.` entirely? Perhaps these are internal details that don't need to be spelled out to readers?

Reply:
I think the `controller broker` will prevent confusion.